controllers/configmap_controller.go

```diff
@@ -579,8 +580,9 @@ func (r *ConfigMapReconciler) ensureWICDRoleBinding() error {
 	}
 	if err == nil {
 		// check if existing RoleBinding's contents are as expected, delete it if not
-		if existingRB.RoleRef == expectedRB.RoleRef &&
-			len(existingRB.Subjects) == len(expectedRB.Subjects) && existingRB.Subjects[0] == expectedRB.Subjects[0] {
+		if reflect.DeepEqual(existingRB.RoleRef, expectedRB.RoleRef) &&
+			len(existingRB.Subjects) == len(expectedRB.Subjects) &&
+			reflect.DeepEqual(existingRB.Subjects[0], expectedRB.Subjects[0]) {
```

Comment:
Do you need to check the len? Would'nt doing `DeepEqual(existingRB.Subjects,expectedRB.Subjects)` check that the length is equal?